route_reflector/decode_cluster_list.c

```diff
@@ -95,12 +95,22 @@ uint64_t decode_cluster_list(args_t *args UNUSED) {
     data = get_arg(ARG_DATA);
     len = get_arg(ARG_LENGTH);
 
-    if (!code || !len || !flags || !data) {
+    COPY_BUFFER(data, *len);
```

Comment:
Same comment as before. If the build is not intended for verification, the macro should not be expanded. How do you manage to prevent this ?